apps/editor.planx.uk/src/routes/views/flowEditor.tsx

```diff
@@ -139,10 +139,6 @@ export const flowEditorView = async (req: NaviRequest) => {
     template,
   });
 
-  // Templated flows require access to an ordered flow
-  // Set this once upstream as it's an expensive operation
-  if (templatedFrom) useStore.getState().setOrderedFlow();
```

Comment:
Having this upstream as part of the route/view/layout means we don't have loaded flow data from ShareDB. I've now moved this to run once the flow data has been fetched.